src/index.js

```diff
@@ -216,19 +232,27 @@ function displayGuestBookings() {
   });
 }
 
-function handleCustomerBooking() {
-  $('#date__picker--js').on('keyup', () => {
-    let dateValue = $('#date__picker--js').val()
-    $('#btn__avalible--rooms--js').on('click', () => {
-      let avalibleRooms = hotel.getAvailableRoomDetailsByDate(dateValue)
-      $('.list__avalible--rooms').html('');
-      avalibleRooms.forEach(room => {
-        let roomsList = $(`<li><h6>Room Number: ${room.number}<br> Room Type: ${room.roomType}<br> Bidet: ${room.bidet}<br> Bed: ${room.bedSize}<br> Number of Beds: ${room.numBeds}<br> Cost: ${room.costPerNight}</h6></li>`);
-        $('.list__avalible--rooms').append(roomsList);
+function filterRoomType() {
+    let type = $('#room__types').find(':selected').val();
+    let filteredRooms = availableRooms.filter(room => room.roomType === type);
+    $('#available__rooms--js').html('');
+    filteredRooms.forEach(room => {
+      const roomItem = $(`<option><h6>Room Number: ${room.number}<br> Room Type: ${room.roomType}<br> Bidet: ${room.bidet}<br> Bed: ${room.bedSize}<br> Number of Beds: ${room.numBeds}<br> Cost: ${room.costPerNight}</h6></option>`);
```

Comment:
I don't know if it's possible to do this without using breaks but it might be worth refactoring so you have multiple <h6> elements instead of one with a break?